acis_thermal_check/utils.py

```diff
@@ -439,10 +448,11 @@ def get_acis_limits(msid):
     import requests
 
     if msid == "fptemp":
-        fp_sens = -118.7
+        fp_sens = -119.5
         acis_i = -112.0
         acis_s = -111.0
-        return fp_sens, acis_i, acis_s
+        acis_hot = -109.0
+        return fp_sens, acis_i, acis_s, acis_hot
```

Comment:
The "various limits" start to get a little confusing for a non-ACIS reviewer, but if you think this is self-documenting seems fine.